Src/Generator/AccessControlGenerator.cs

```diff
@@ -200,9 +200,12 @@ static Allow()
                  {
                      foreach (var f in typeof(Allow).GetFields(BindingFlags.Public | BindingFlags.Static))
```

Comment:
wouldn't just filtering out the `Descriptions` dictionary from here be a better alternative?
i.e. the generated `Allow.g.cs` has the following:
```cs
#region DESCRIPTIONS
    public static Dictionary<string, string> Descriptions = new()
    {
        //Customers_Create
        { "JHH", "Permission for creating a new customer in the system." },
        //Sales_Order_Create
        { "LYO", "Allows creation of orders by anybody who has this permission code." },
    };
#endregion
```
which is where the rogue `"System.Collections.Generic.Dictionary`2[System.String,System.String]"` entry comes from. so i think just filtering out this field by name `Descriptions` should be the way to go.

what do you think?

Reply:
Yeah that would work just fine. The reason I chose to filter strings in general to be included is so if in the future some other similar functionality gets added to the genrator then this issue wouldn't resurface. But then again it could also cause issues in the future that don't exist now... so it's hard to decide. Now I think either soultion is fine.

Reply:
Or if someone choses to add something similar to the class by extending it. I haven't check yet wheter that would cause the same tho.

Reply:
we can decorate the `Descriptions` property with the existing attribute `[HideFromDocs]` and use that to exclude the field. if we filter by attribute, users can also use the same strategy (decorate with [HideFromDocs]) if you they wanna add some public static fields/properties to their partial class and it will be automatically excluded. sounds like a plan?

Reply:
Yeah that sounds great. I can make the changes in about an hour.

Reply:
Alright , thanks for the suggestions.

Reply:
Pushed a new commit, that should do it.